src/game/client/gameclient.cpp

```diff
@@ -1373,6 +1378,14 @@ void CGameClient::OnNewSnapshot()
 			{
 				m_Snap.m_paFlags[Item.m_ID%2] = (const CNetObj_Flag *)pData;
 			}
+			else if (Item.m_Type == NETOBJTYPE_HARPOONDRAGPLAYER)
+			{
+				if (m_Snap.HarpoonDragCount < 64) //64 packets are enough. Surely nobody is gonna send more than this
```

Comment:
I have to say this immediately. Dont drop apckets when it's "too much". mods can always use it and this is a great thing for mod nakers. Would be cool if you can remove this :D